integration_tests/src/main/python/data_gen.py

```diff
@@ -330,18 +330,26 @@ def start(self, rand):
 
 class RepeatSeqGen(DataGen):
     """Generate Repeated seq of `length` random items"""
```

Comment:
Nit: Should update the docstring to explain the new mode and args required (or disallowed) for the different forms.